tests/core/test_dataset.py

```diff
@@ -1030,14 +1103,14 @@ def test_copy_structure(grid):
 
 
 def test_copy_structure_self(datasets):
-    for dataset in datasets:
-        copied = dataset.copy()
-        assert copied is not dataset
+    for dset in datasets:
```

Comment:
I prefer to keep it as a name `dataset` .

Reply:
Done in https://github.com/pyvista/pyvista/pull/7252/commits/ef9f03e2b438c2d70baf63743657bda67f5e7742